src/components/UI/homepage/QuickLinks.tsx

```diff
@@ -0,0 +1,141 @@
+import { Box, Grid, GridItem, Link, Stack, Text } from '@chakra-ui/react';
+import { FC } from 'react';
+import NextLink from 'next/link';
+
+import { CONTRIBUTING_PAGE, DOCS_PAGE, FAQ_PAGE } from '../../../constants';
+
+export const QuickLinks: FC = () => {
+  return (
+    <Stack border='2px solid' borderColor='brand.light.primary'>
+      <Stack p={4} borderBottom='2px solid' borderColor='brand.light.primary'>
+        <Box as='h2' textStyle='h2'>
+          Quick Links
+        </Box>
+      </Stack>
+
+      <Grid templateColumns='repeat(2, 1fr)' sx={{ mt: '0 !important' }}>
+        {/* get started */}
+        <GridItem
+          borderRight='2px solid'
+          borderBottom='2px solid'
+          borderColor='brand.light.primary'
+        >
+          <Stack p={4} h='100%'>
+            <Text textStyle='quick-link-text' mb={-1}>
+              Don&apos;t know where to start?
+            </Text>
+
+            <Text textStyle='quick-link-text'>We can help.</Text>
+          </Stack>
+        </GridItem>
+        <GridItem borderBottom='2px solid' borderColor='brand.light.primary'>
+          <NextLink href={`${DOCS_PAGE}/getting-started`} passHref>
+            <Link _hover={{ textDecoration: 'none' }}>
+              <Stack
```

Comment:
Made a cleanup task for this, but not a blocker.

https://github.com/ethereum/geth-website/issues/14